docs/dub-reference/package_settings.md

```diff
@@ -27,7 +27,7 @@ Technically uppercase characters are parsed, but not valid for publishing and wi
     name "openssl"
     ```
 
-    Arguments: `"<name>"`
+    Syntax: name "<package-name>"
```

Comment:
I think the new names and syntax is good, but please keep the code in inline code blocks (\`quoted\` text)

this applies to all remaining changes